### PR TITLE
chore: move some hooks from portal to theme

### PR DESCRIPTION
**Pull request by slavikbez**
## What/Why/How?

Move some of the hooks from portal to theme.

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "peerDependencies": {
+    "lodash.throttle": "^4.1.1",
```
Comment (volodymyr-rutskyi) [inline package.json:13]: Is there a reason you didn't commit changes to package.lock.json?

File: src/TableOfContent/TableOfContent.tsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import styled from 'styled-components';
+
 import type { MdHeading } from '@portal/types';
 import { mediaQueries } from '@portal/media-css';
-import { useFullHeight, useActiveHeading } from '@portal/hooks';
+import { useFullHeight } from '@theme/hooks/useFullHeight';
```
Comment (volodymyr-rutskyi) [inline src/TableOfContent/TableOfContent.tsx:6]: Using default exports would make imports a little bit nicer. Just a personal preference though :)

File: package.json (hunk 1 above)
Comment (slavikbez) [reply package.json:13]: This is a peer dependency

File: src/TableOfContent/TableOfContent.tsx (hunk 2 above)
Comment (slavikbez) [reply src/TableOfContent/TableOfContent.tsx:6]: good idea 👍

Comment (ozapotichnyi) [reply src/TableOfContent/TableOfContent.tsx:6]: > Do not use default exports. Importing modules must give a name to these values, which can lead to inconsistencies in naming across modules.
https://google.github.io/styleguide/jsguide.html#es-module-exports

**Review by ozapotichnyi — APPROVED** [review]

**Pull request merged**